bin/compile-requirements.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+set -exo pipefail
+
+export CUSTOM_COMPILE_COMMAND="make compile-requirements"
+
+# We need this installed, but we don't want it to live in the main requirements
+# We will need to periodically review this pinning
+pip install --upgrade pip-tools==6.4.0  # needs at least this version to build
+pip install pip-compile-multi
+
+if [[ "$1" == "--upgrade" ]]; then
+    pip-compile-multi \
+        --generate-hashes prod \
+        --generate-hashes dev \
+        --generate-hashes docs \
+        --header=/app/bin/pip-compile-multi-header-message.txt \
+        --upgrade
```

Comment:
Nice catch. I'll make compile into `--no-upgrade`. The reason I kept them separate so that it's possible to add a new dep without triggering an upgrade of all the others. HOWEVERRRRR, because we hard-pin everything, `--upgrade` will basically do nothing unless we soft-pin/range-pin things... What do you think?

Reply:
I like the simplicity of having a single call if they are effectively the same.

But, correct me if I'm wrong, if we always did `--upgrade` and subdeps had an update, they would get pulled in as well, right?

Reply:
See earlier comment - am ditching `upgrade requirements` and we'll just have `compile requirements`